src/utils.js

```diff
@@ -270,9 +270,10 @@ function printLogs (activation, strip, logger) {
  *    ['pkg1/action'] = logs of action 'action' under package 'pkg1'
  *    [] = logs of all actions in the namespace
  * @param {boolean} strip if true, strips the timestamp which prefixes every log line
+ * @param {boolean} cleanLogs if false, print old logs occurrence, if true, show only new logs occurrence.
  * @param {number} startTime time in milliseconds. Only logs after this time will be fetched
  */
-async function printFilteredActionLogs (runtime, logger, limit, filterActions = [], strip = false, startTime = 0) {
+async function printFilteredActionLogs (runtime, logger, limit, filterActions = [], strip = false, cleanLogs = false, startTime = 0) {
```

Comment:
tests fail, specifically branch coverage. Run `npm test` locally  to see the table. Lines 276 and 313 in `utils.js`